src/components/Articles/FeaturedArticles/FeaturedArticles.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import 'dotenv/config';
+import placeholder from '../../../assets/images/placeholder.png';
+
+export default class FeaturedArticles extends Component {
```

Comment:
Can you explain to me the use of this component? thanks

Reply:
I am not using it right now. 
It will serve this component 
<img width="1060" alt="Screenshot 2019-07-06 at 03 03 27" src="https://user-images.githubusercontent.com/10960862/60749462-bd3a3800-9f9a-11e9-93d4-e2186be9149e.png">